extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -27,25 +30,29 @@ export class Entitlement {
    * @return {!Entitlement}
    */
   static empty(service) {
-    return new Entitlement(
-        /* source */ '',
-        /* raw */ '',
-        service,
-        /* products */ [],
-        /* subscriptionToken */ null,
-        /* loggedIn */ false);
+    return new Entitlement({
```

Comment:
Use as much as default state in this call as possible.

Reply:
Done